members/MEL000279.yaml

```diff
@@ -0,0 +1,57 @@
+bioguide: MEL000279
+contact_form:
+  method: post
+  action:
+  steps:
+    - visit: "https://legislature.maine.gov/housedems/contactus/index.html"
+    - select:
+      - name: Representative
+        selector: select#select
+        value: Representative Heidi Brooks
```

Comment:
`required: true`?

Reply:
question: is it necessary to add the `required` step even if the value (in this case the Rep's name) is hardcoded to the yaml?

Reply:
iirc either simplebrowser or phantom skips non required fields either on the initial post or if there's a failure. We need this field present if we're going to submit to the correct rep, so it should be `required: true` IMO

Reply:
ohh good to know. Just added `required: true` for both Representative and Topic